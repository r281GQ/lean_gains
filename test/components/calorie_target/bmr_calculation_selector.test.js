import React from 'react';
import { shallow } from 'enzyme';
import BMRCalculationSelector from './../../../src/components/calorie_target/bmr_calculation_selector';

describe('BMRCalculationSelector test', () => {
  it('should render', () => {
    const component = shallow(<BMRCalculationSelector />);
    expect(component).toHaveLength(1);
    expect(component).toMatchSnapshot();
  });
});
