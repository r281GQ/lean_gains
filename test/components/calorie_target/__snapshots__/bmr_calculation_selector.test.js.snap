// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BMRCalculationSelector test should render 1`] = `
<Field
  component={[Function]}
  floatingLabelText="Select your bmr calculation method"
  fullWidth={true}
  name="bmrCalculationMethod"
>
  <MenuItem
    anchorOrigin={
      Object {
        "horizontal": "right",
        "vertical": "top",
      }
    }
    checked={false}
    desktop={false}
    disabled={false}
    focusState="none"
    insetChildren={false}
    primaryText="Based on weight (Harris-Benedict)"
    targetOrigin={
      Object {
        "horizontal": "left",
        "vertical": "top",
      }
    }
    value="harris-benedict"
  />
  <MenuItem
    anchorOrigin={
      Object {
        "horizontal": "right",
        "vertical": "top",
      }
    }
    checked={false}
    desktop={false}
    disabled={false}
    focusState="none"
    insetChildren={false}
    primaryText="Based on lean body mass (Katch-Mcardle)"
    targetOrigin={
      Object {
        "horizontal": "left",
        "vertical": "top",
      }
    }
    value="katch-mcardle"
  />
</Field>
`;
